python/ez_setup.py

```diff
@@ -1,284 +1,332 @@
-#!python
-
-# This file was obtained from:
-#   http://peak.telecommunity.com/dist/ez_setup.py
-# on 2011/1/21.
-
+#!/usr/bin/env python
```

Comment:
To add just a bit more detail, the file should (if possible) go in a directory like:

```
  <base>/third_party/python_ez_setup/ez_setup.py
  <base>/third_party/python_ez_setup/README.google
  <base>/third_party/python_ez_setup/LICENSE
```

For a model, see: https://github.com/google/upb/tree/master/third_party/lunit

If this is too much to figure out for this CL, let's just leave the update of ez_setup.py out of this PR and we'll figure this part out later.